scripts/Inelastic/IndirectReductionCommon.py

```diff
@@ -4,10 +4,16 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
-from mantid.simpleapi import AppendSpectra, DeleteWorkspace, Load, plotSpectrum
+from mantid.simpleapi import AppendSpectra, DeleteWorkspace, Load
 from mantid.api import AnalysisDataService, WorkspaceGroup, AlgorithmManager
 from mantid import mtd, logger, config
 
+try:
+    import plotSpectrum
```

Comment:
I didn't spot this but I think this import is wrong? Should it not be `from mantid.simpleapi import plotSpectrum`?

It looks like if you start Workbench at the moment you get: `plotSpectrum not available` printed.

Reply:
I've created #34929 to track it. Would it be possible to get this fixed as a high priority before we put the beta out?